Tribes/src/app/notification/notification.component.css

```diff
@@ -0,0 +1,12 @@
+.container-fluid{
+    background-color: #263740;
+    border-radius: 20px;
+    padding-top: 10px;
+    color: whitesmoke;
+    list-style-type: none;
+}
+
+img{
+    width: 45px;
+    padding: 5px;
+}
```

Comment:
It says no new line at the end of code. Did you run it past linter? Just asking, you may have...